src/baselitex/constr/arty-a35t-nextpnr-xilinx.xdc

```diff
@@ -0,0 +1,310 @@
+set_property LOC H5 [get_ports {led[0]}]
```

Comment:
Why is the xdc specific to nextpnr?

Reply:
There seems to be an issue with reading XDC constraints related to the iobank (to get the INTERNAL_VREF). That line has been commented out for the nextpnr-xilinx test.